package.json

```diff
@@ -57,16 +56,10 @@
     "imagemin-jpegtran": "5.0.2",
     "imagemin-optipng": "5.2.1",
     "imagemin-svgo": "5.2.4",
-    "is-reachable": "2.4.0",
     "lightbox2": "2.10.0",
-    "localtunnel": "1.9.0",
     "lodash.throttle": "4.1.1",
     "postcss-unmq": "1.0.2",
-    "psi": "3.1.0",
-    "require-dir": "1.0.0",
     "sauce-connect-tunnel": "0.2.1",
```

Comment:
This can probably be moved as well.

Reply:
It's used by the acceptance tests, which are run in the production environment, so has to stay in dev-deps.

Reply:
Err I mean deps

Reply:
Wait… what test commands are run on Jenkins?

Reply:
Okay, this is updated to not have the test task in the production environment. The discussion can continue in https://github.cfpb.gov/CFGOV/platform/issues/2709 to set Travis to the production environment + tests.